packages/fractal-page-object/API.md

```diff
@@ -115,14 +117,14 @@ matching this page object's query if this page object does not have an
 index, or the `index`th matching DOM element if it does have an index
 specified.
 
-Type: ([Element][24] | null)
+Type: (ElementType | null)
 
 ### elements
 
 This page object's list of matching DOM elements. If this page object has
 an index, this property will always have a length of 0 or 1.
 
-Type: [Array][25]<[Element][24]>
+Type: [Array][23]\<ElementType>
 
 ## selector
 
```

Comment:
Suggestion: I think the docs here could be updated to include the generic types variables, especially since there are two now.